src/email.ts

```
@@ -1,40 +1,29 @@
 import { extract as parseRawEmail } from 'letterparser';
-import { splitEllipsis } from './splitMessage';
-
-const DISC_MAX_LEN = 2000;
 
 export async function email(message: any, env: any, ctx?: any): Promise<void> {
   const url = env.DISCORD_WEBHOOK_URL;
+  const DEFAULT_MAX_LENGTH = 524288; // 512KB
+  const maxLenFromEnv = parseInt(env.MAX_LEN);
+  const MAX_LEN: number = !isNaN(maxLenFromEnv) && maxLenFromEnv > 0 
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)